templates/data/mysql.html

```diff
@@ -480,9 +536,11 @@ <h3 class="p-heading--5">From our blog</h3>
 
 <section class="p-section--deep">
   <div class="row">
-    <hr class="p-rule u-hide--small">
-    <div class="col-6">
-      <p>"MySQL" is a trademark or registered trademark of Oracle. Other trademarks are property of their respective owners.</p>
+    <hr />
+    <div class="u-text--muted col-start-large-7 col-6">
```

Comment:
In the copy doc it was asked to leave the footer as is. It was full width, not on the right

Reply:
I tried doing it with the full width and it would still use the 6-cols only and not the entire row.
So I did how we are doing on other pages of same bubble like canonical.com/data/mongodb

Reply:
I believe design should also confirm this @mattea-turic